File: app/src/main/java/app/rental/utsavstha/rentalapp/interfaces/CallBacks.java
package app.rental.utsavstha.rentalapp.interfaces;

/**
 * Created by utsavstha on 8/19/17.
 */

public interface CallBacks {
    interface ListingsAdapterCallBack{
        void onListingClicked(int position);
    }

    interface AlertsAdapterCallBack{
        void onAlertClicked(int position);
    }

}

File: app/src/main/java/app/rental/utsavstha/rentalapp/interfaces/CallBacks.kt
package app.rental.utsavstha.rentalapp.interfaces

import android.widget.ImageView

/**
 * Created by utsavstha on 8/19/17.
 */

interface CallBacks {
    interface ListingsAdapterCallBack {
        fun onListingClicked(position: Int)
    }

    interface AlertsAdapterCallBack {
        fun onAlertClicked(position: Int)
    }

    interface PostAdsCallBack {
        fun onImageClicked(position: Int, imageView: ImageView)
    }

}
